test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -41,7 +42,10 @@ public class ContentResultTest
             await contentResult.ExecuteResultAsync(actionContext);
 
             // Assert
-            Assert.Equal("text/plain; charset=utf-7", httpContext.Response.ContentType);
+            Assert.True(MediaTypeComparisons.AreEquivalent(
```

Comment:
String equality is fragile, we don't care about non significant whitespaces. I move the class to TestCommon, I'll rename it and follow the TypeAssert ... pattern